try {
  noExistentFunction();
}

catch(error) {
  console.log(error)
}
